javascript/apis/video-audio/finished/index.html

```diff
@@ -1,27 +1,30 @@
 <!DOCTYPE HTML>
 <html lang="en-gb">
   <head>
-    <meta charset="utf-8">
+  	<meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
   	<title>Video player example</title>
   	<link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Alignment looks incorrect here… You should run Prettier on it.
```suggestion
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Video player example</title>
    <link rel="stylesheet" type="text/css" href="style.css">
```